cli/src/main/java/com/devonfw/tools/ide/tool/intellij/Intellij.java

```diff
@@ -33,8 +35,21 @@ public boolean install(boolean silent) {
   @Override
   public void installPlugin(PluginDescriptor plugin) {
 
-    // TODO Auto-generated method stub
-
+    // no override needed
   }
```

Comment:
maybe you could just remove the method if it's not needed.

Reply:
This method must be declared.